app/views/shared/_header.html.erb

```diff
@@ -22,6 +22,7 @@
                         <%= current_user.name %>
                     </a>
                     <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                        <%= link_to "Dashboard", "/dashboard/#{session[:user_id]}",class: "dropdown-item" %>
```

Comment:
We dont need to put the id in the dashboard url. We can just say '/dashboard'. So please remove the user_id. And please use dashboard_path